src/components/Header/Header.js

```diff
@@ -241,13 +241,12 @@ function Header() {
         .filter(
           ({ href }) =>
             href !== '/' &&
-            href !== '/register' &&
             href !== '/notifications' &&
             href !== '/login?logout=true' &&
             href !== '/settings'
         )
         .map(({ id, href, label }) => {
-          let color = label === 'Donate' ? '#B9EBEC' : 'white';
+          let color = label === 'Join Us' ? '#B9EBEC' : 'white';
```

Comment:
make it a const if you're not modifying the value anywhere else

Reply:
resolved in 750b857